### DISCUSSION TITLE
Does a company using next-auth have an issue with gdpr?

### DISCUSSION THREAD

**Discussion by vincent-thomas** [nextauthjs/next-auth#4812] [Help]
Hi! We are looking into using next-auth for our authentication handler and will be using prisma adapters. Is the prisma adapter gdpr compliant and are we liable for gdpr fines if data gets exposed to the public?

**Comment by eug-vs** [comment]
I'm also confused. If I we only use cookies for auth on our own website, such cookies can only be used to identify a user in our application. We might as well use JWT instead, and then we certainly don't need any "jwt consent". It's just a detail of implementation. Maybe I'm wrong and don't know something, please correct me.
